src/Eagle.ts

```diff
@@ -44,6 +44,7 @@ import {Edge} from './Edge';
 import {Field} from './Field';
 import {FileInfo} from './FileInfo';
 import {Setting} from './Setting';
+import { collapseTextChangeRangesAcrossMultipleVersions } from "typescript";
```

Comment:
Do we need this import? The function "collapseTextChangeRangesAcrossMultipleVersions" doesn't seem to be used in Eagle.ts. If not required, you could remove it. If it is required, then please add a comment with the reason and/or a URL where the fix was found.

Reply:
i dont remember adding this line. ill ask Andreas if its from him